lib/srv/discovery/common/database.go

```diff
@@ -1290,6 +1328,16 @@ func ExtraElastiCacheLabels(cluster *ectypes.ReplicationGroup, tags []ectypes.Ta
 	return addLabels(labels, libcloudaws.TagsToLabels(tags))
 }
 
+// ExtraElastiCacheServerlessLabels returns a list of extra labels for provided
+// ElastiCacheServerless cluster.
+func ExtraElastiCacheServerlessLabels(cache *ectypes.ServerlessCache, tags []ectypes.Tag) map[string]string {
+	labels := make(map[string]string)
+	labels[types.DiscoveryLabelEngine] = aws.ToString(cache.Engine)
+	labels[types.DiscoveryLabelEngineVersion] = aws.ToString(cache.FullEngineVersion)
```

Comment:
any easy way to find the VPC ID? the dashboard does show it. worse case we can get it by describing subnet or security group.